WordPress/gutenberg
# Iterate zoom out shuffle into a more visual control

## What?
Resolves: https://github.com/WordPress/gutenberg/issues/66105


This PR will update the `shuffle` feature for patterns with a more visual control that shows the previews of some available patterns from the same pattern category.

I [commented](https://github.com/WordPress/gutenberg/issues/66105#issuecomment-2418699620) the below in the issue: 

> It's worth noting that we have other similar patterns lists like:
>1. Quick inserter previews
>2. `Replace` in blocks like Query Loop (bigger dialog)

> This issue suggests yet another design. Should we proceed with the new design and see if can consolidate designs or this is considered the best one for this specific functionality?

--cc @WordPress/gutenberg-design 

Currently I'm using the `BlockPatternsList` - same with quick inserter.



## Testing Instructions
1. Check that there is no `shuffle` button and in its place we render the `change design` one.
2. Changing patterns should work as expected.



## Screenshots or screencast 
TBD
